src/sections/account/Account.tsx

```diff
@@ -0,0 +1,58 @@
+import { useTranslation } from 'react-i18next'
+import { useSearchParams } from 'react-router-dom'
+import { Tabs } from '@iqss/dataverse-design-system'
+import { useLoading } from '../loading/LoadingContext'
+import { AccountHelper } from './AccountHelper'
+import { ApiTokenSection } from './api-token-section/ApiTokenSection'
+import { BreadcrumbsGenerator } from '../shared/hierarchy/BreadcrumbsGenerator'
+import { UpwardHierarchyNodeMother } from '../../../tests/component/shared/hierarchy/domain/models/UpwardHierarchyNodeMother'
+import styles from './Account.module.scss'
+import { useEffect } from 'react'
+
+const tabsKeys = AccountHelper.ACCOUNT_PANEL_TABS_KEYS
+
+export const Account = () => {
+  const { t } = useTranslation('account')
+  const { setIsLoading } = useLoading()
+  const [searchParams] = useSearchParams()
+  const defaultActiveTabKey = AccountHelper.defineSelectedTabKey(searchParams)
+
+  const rootHierarchy = UpwardHierarchyNodeMother.createCollection({
```

Comment:
The name and alias of the root collection is not guaranteed to be 'root', so I think we need to replace this with a call to get the root collection from the API.  There is a special keyword, ':root', that returns the root collection: https://guides.dataverse.org/en/latest/api/native-api.html#view-a-dataverse-collection.  So we should also change the default ROOT_COLLECTION_ALIAS to ':root'.

Reply:
hmm thats true, interesting, I think the change will require a completely separate issue (maybe a js-dataverse use case first), as it will be quite large, I think.
We rely on a Root word and root alias in several places of the application at this point.

Reply:
Now thinking... we will also need to use that "dynamic" root alias in the URL paths and not the hardcoded root, quite a challenge for an SPA, so we really need to first create a use case for it and then make all the necessary changes in the SPA, maybe not a priority right now? but for sure an really important thing.

Reply:
Yes, you're right I think it makes sense to add another issue for that.

Reply:
I added an issue for that, here: https://github.com/IQSS/dataverse-frontend/issues/489